docs/verifiable-document/dns-proof.md

```diff
@@ -4,8 +4,60 @@ title: Configuring DNS
 sidebar_label: Configuring DNS
 ---
 
-Overview
+Every OA document's provenance can be verified and traced back to it's creator or issuer. This is achieved by embedding an `identityProof` in the document which serves as a claim of identity. During the verification phase, the claim is checked against external records.
 
-## Header
+In this guide, we will bind the document issuer's identity to a valid domain name. This domain will be displayed as issuer every time the document is rendered in a complaint viewer.
 
-TBD
+## Example
+
+![Example Issuer Identity](/docs/verifiable-document/dns-proof/example.png)
+
+In this example, the document's issuer is bound to `DEMO.TRADETRUST.IO`.
+
```

Comment:
Wondering if we shouldn't stick with open attestation for example instead of tradetrust

Reply:
Need to change the other image also